lib/rules/function-calc-no-unspaced-operator/__tests__/index.mjs

```diff
@@ -225,17 +229,13 @@ testRule({
 			code: 'a { top: calc(10px/var(--foo)); }',
 		},
 		{
-			code: 'a { padding: calc(); }',
+			code: 'a { top: calc(10px*var(--foo, 10px 20px)); }',
 		},
 		{
-			code: 'a { font-size: clamp(1rem, 2.5vw, 1rem+1rem); }',
-			description:
-				'multiple argument: functions that accept more than one argument are not supported yet',
+			code: 'a { top: calc(10px*var(--foo, 10px+20px)); }',
 		},
 		{
-			code: 'a { width: min(25vw+25vw); }',
-			description:
-				'single argument: functions that accept more than one argument are not supported yet',
+			code: 'a { padding: calc(); }',
```

Comment:
Can you keep the original order of the examples to help with the review?

Reply:
Let me check!

Reply:
Should be better now.

Reply:
e.g.  l635